### ISSUE TITLE
flit dont' like when modules have no docs.

### ISSUE THREAD

**Issue by Carreau** [takluyver/flit#11]
```
$flit wheel
Fetching list of valid trove classifiers                          I-flit.inifile
Starting new HTTPS connection (1): pypi.python.org  I-requests.packages.urllib3.connectionpool
Traceback (most recent call last):
  File "/usr/local/bin/flit", line 11, in <module>
    sys.exit(main())
  File "/Users/bussonniermatthias/Library/Python/3.5/lib/python/site-packages/flit/__init__.py", line 57, in main
    verify_metadata=args.verify_metadata).build()
  File "/Users/bussonniermatthias/Library/Python/3.5/lib/python/site-packages/flit/wheel.py", line 31, in __init__
    self.metadata = common.make_metadata(self.module, self.ini_info)
  File "/Users/bussonniermatthias/Library/Python/3.5/lib/python/site-packages/flit/common.py", line 148, in make_metadata
    md_dict.update(get_info_from_module(module))
  File "/Users/bussonniermatthias/Library/Python/3.5/lib/python/site-packages/flit/common.py", line 38, in get_info_from_module
    docstring_lines = m.__doc__.splitlines()
AttributeError: 'NoneType' object has no attribute 'splitlines'
```

**Comment by takluyver** [comment]
Quite deliberately - the README says: "Make sure that foobar’s docstring starts with a one-line summary of what the module is, and that it has a __version__"

In both cases, the error should be more explanatory, though.

**Comment by Carreau** [comment]
closed by #16
